### PR TITLE
Make Substratum great again

### PR DESCRIPTION
**Pull request by sirmordred**
More robust

### CODE REVIEW

**Comment by sirmordred** [comment]
Any review?

**Comment by nicholaschum** [comment]
Please give us some time, I'll check it out when I get home.

File: app/src/main/java/projekt/substratum/common/commands/FileOperations.java [hunk 1]
```diff
@@ -328,6 +334,9 @@ public static void move(Context context, String source, String destination) {
             } catch (InterruptedException e) {
                 Thread.interrupted();
             }
+            if (!file.exists()) {
+                move(source, destination);
```
Comment (nicholaschum) [inline app/src/main/java/projekt/substratum/common/commands/FileOperations.java]: So you are checking if things aren't moving by ThemeInterfacer, you will ask the app too? 

This should be impossible to reach as we use the ThemeInterfacer to move files that third party apps cannot move, such as /data/system/theme folder.

File: app/src/main/java/projekt/substratum/common/commands/FileOperations.java [hunk 2]
```diff
@@ -408,8 +409,8 @@ public static boolean copyFileOrDir(AssetManager assetManager, String listDir,
 
     private static boolean copyFile(AssetManager assetManager, String filename,
                                     String destination, String remember) {
-        InputStream inputStream;
-        OutputStream outputStream;
+        InputStream inputStream = null;
```
Comment (nicholaschum) [inline app/src/main/java/projekt/substratum/common/commands/FileOperations.java:420]: In Android Studio, does lint complain that these "= null" checks are redundant?

**Review by nicholaschum — CHANGES_REQUESTED** [review]

File: app/src/main/java/projekt/substratum/common/commands/FileOperations.java (hunk 1 above)
Comment (sirmordred) [reply app/src/main/java/projekt/substratum/common/commands/FileOperations.java]: Done

File: app/src/main/java/projekt/substratum/common/commands/FileOperations.java (hunk 2 above)
Comment (sirmordred) [reply app/src/main/java/projekt/substratum/common/commands/FileOperations.java:420]: it didnt give any warning about redundancy of these checks but
i can remove them if you want although

**Pull request merged**